src/server/ntpcontroller.cpp

```diff
@@ -77,12 +96,7 @@ void NtpController::setConnmanProperty(QString key, QString value)
     } else {
         log_debug("Set %s property %s to value %s",
                   QString(CONNMAN_INTERFACE).toStdString().c_str(),
-                  key.toStdString().c_str(),
-                  value.toStdString().c_str());
+                  reply.argumentAt(0).toString().toStdString().c_str(),
+                  reply.argumentAt(1).toString().toStdString().c_str());
     }
```

Comment:
Yep, doing an update of sb2 and running ldconfig in the target fixed things for me.